lib/node_modules/@stdlib/blas/ext/base/ndarray/dsortins/test/test.js

```diff
@@ -0,0 +1,83 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Float64Array = require( '@stdlib/array/float64' );
+var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+var getData = require( '@stdlib/ndarray/data-buffer' );
+var getDtype = require( '@stdlib/ndarray/dtype' );
+var getStrides = require( '@stdlib/ndarray/strides' );
+var getOffset = require( '@stdlib/ndarray/offset' );
+var getShape = require( '@stdlib/ndarray/shape' );
+var ndarray = require( '@stdlib/ndarray/base/ctor' );
+var dsortins = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof dsortins, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function sorts a one-dimensional ndarray (increasing order) using insertion sort.', function test( t ) {
```

Comment:
```suggestion
tape( 'the function sorts a one-dimensional ndarray (increasing order) using insertion sort', function test( t ) {
```

@MeKaustubh07 This has come up in a few of your PRs. Please avoid including periods at the end of test descriptiosn.

Reply:
Okay!, I will keep that in mind.